api-test/src/main/resources/signup/AuthenticateUserPasswordNegTC/AuthenticateUserPasswordNegTC.yml

```
@@ -1002,6 +1003,7 @@ AuthenticateUserPasswordNegTC:
       allowedErrorCodes: invalid_challenge_format,invalid_auth_factor_type,invalid_challenge,auth_failed
       checkErrorsOnlyInResponse: true
       validityCheckRequired: true
+      additionalDependencies: TC_ESignet_VerifyChallengeForResetPassword_01,TC_ESignet_RegisterUser_01,TC_ESignet_ResetPassword_01,TC_ESignet_GenerateChallenge_01,TC_ESignet_VerifyChallenge_01,TC_ESignet_GenerateChallengeForResetPassword_01
       allowedErrorCodes: IDA-PSD-001,invalid_challenge,auth_failed
```

YAMLlint (1.38.0)

[error] duplication of key "allowedErrorCodes" in mapping

(key-duplicates)